samples/ResourceOwnerPasswordFlow/DeveloperNotes/ConformanceProfiles/conformance-profiles.txt

```diff
@@ -0,0 +1,4 @@
+﻿http://openid.net/wordpress-content/uploads/2015/03/OpenID-Connect-Conformance-Profiles.pdf
```

Comment:
I know the tool, but I haven't taken the time to try it :smile: 

That said, I'm not sure the basic tests will pass if you don't use one of the standard flows (code, hybrid, implicit). And I guess there's no conformance test for the ROPC flow, which is not an OpenID Connect thing.